collections-arrays/ArrayPool/ArrayPool.Benchmark/Benchmark.cs

```diff
@@ -0,0 +1,33 @@
+﻿using BenchmarkDotNet.Attributes;
+using System.Buffers;
+
+namespace ArrayPool.Benchmark;
+
+[MemoryDiagnoser]
+public class Benchmark
+{
+    [Params(100, 1000, 10000, 100000)]
+    private int ArraySize { get; set; }
+
+    private ArrayPool<int>? arrayPool;
```

Comment:
Private fields names should start with an underscore, so this should be `_arrayPool` instead of `arrayPool`

Reply:
Fixed!